doc/nrf/includes/apply_cmake_variable.txt

```diff
@@ -0,0 +1,36 @@
+.. |makevarformatstart| raw:: html
+
+   <strong class="makevar">
+
+.. |makevarformatend| raw:: html
+
+   </strong>
+
+.. |fileformatstart| raw:: html
+
+   <code class="file docutils literal notranslate"><span class="pre">
+
+.. |fileformatend| raw:: html
+
+   </span></code>
+
+To enable |variable_feature|, build the application with the |makevarformatstart|\ |makevar|\ |makevarformatend| variable set to |fileformatstart|\ |cmake_file_name|\ |fileformatend|.
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
```

Comment:
This file might still change in the next PRs that tackle other variables, other than EXTRA_CONF_FILE. As things stand now, however, I will not make this change, because the VS Code extension is the recommended IDE (see https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation/install_ncs.html#installing-the-ncs ) and goes first in all tab constructs of this sort.

Reply:
That's what I was afraid of 😄

Reply:
Guess I'm also biased here, but I agree with @trantanen that it's seems unnecessary to have VS Code here at all. The information in the VS Code tab is generic and intuitive for developers to figure out themselves after they've done it once, so wouldn't it be enough if it was only explained in the link we provide for more info or one of the beginner guides? For command line however it can be difficult to remember the variable names, and it's handy to just copy-paste the command.

I'm curious in what circumstances you'd need to specify CMake variables when using VSCode. That sounds like an extreme rare case, and most likely something that should be addressed in VS Code instead of complicating our docs everywhere we mention overlays. If not, then perhaps we can only include the VS Code tab where it's strictly necessary instead of complicating all other use-cases?